server/services/licenceService.ts

```diff
@@ -89,6 +89,16 @@ export class LicenceService {
     }
   }
 
+  getCurfewAddressReviewVersion(licence: Licence): AddressReview['version'] {
```

Comment:
if version already set, use it. 
if curfew address section started but not completed, set version to 1.
For all other cases, set to version 2